servers/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -15,1704 +15,183 @@
+    mTachyonConf = Preconditions.checkNotNull(tachyonConf);
+    mUFSDataFolder = mTachyonConf.get(Constants.UNDERFS_DATA_FOLDER, Constants.DEFAULT_DATA_FOLDER);
+    mStartTimeMs = System.currentTimeMillis();
   }
 
   /**
-   * Returns a list of the given folder's children, recursively scanning subdirectories. It adds the
-   * parent of a node before adding its children.
+   * Gets the capacity of the whole system.
    *
-   * @param inodeFolder The folder to start looking at
-   * @return a list of the children inodes.
+   * @return the system's capacity in bytes.
    */
-  private List<Inode> getInodeChildrenRecursive(InodeFolder inodeFolder) {
-    synchronized (mRootLock) {
-      List<Inode> ret = new ArrayList<Inode>();
-      for (Inode i : inodeFolder.getChildren()) {
-        ret.add(i);
-        if (i.isDirectory()) {
-          ret.addAll(getInodeChildrenRecursive((InodeFolder) i));
-        }
-      }
-      return ret;
-    }
+  public long getCapacityBytes() {
+    return mBlockMaster.getCapacityBytes();
   }
 
   /**
-   * Get Journal instance for MasterInfo for Unit test only
+   * Gets the list of block info of an InodeFile determined by path.
    *
-   * @return Journal instance
+   * @param path path to the file
+   * @return The list of the block info of the file
+   * @throws InvalidPathException when the path is invalid
+   * @throws FileDoesNotExistException when the file does not exist
    */
-  public Journal getJournal() {
-    return mJournal;
+  public List<BlockInfo> getBlockInfoList(TachyonURI path)
+      throws InvalidPathException, FileDoesNotExistException {
+    return mFileSystemMaster.getBlockInfoList(path);
   }
 
   /**
-   * Get the master address.
+   * Gets the list of blocks of an {@link InodeFile} determined by the file id.
    *
-   * @return the master address
+   * @param fileId id of the file
+   * @return The list of the blocks of the file
+   * @throws InvalidPathException when the path is invalid
+   * @throws FileDoesNotExistException when the file does not exist
    */
-  public InetSocketAddress getMasterAddress() {
-    return mMasterAddress;
+  public List<FileBlockInfo> getFileBlockList(long fileId)
+      throws InvalidPathException, FileDoesNotExistException {
+    return mFileSystemMaster.getFileBlockInfoList(fileId);
   }
 
   /**
-   * Get the MasterSource instance
+   * Gets the dependency info associated with the given id.
    *
-   * @return the MasterSource instance
+   * @param dependencyId id of the dependency
+   * @return the dependency info
+   * @throws DependencyDoesNotExistException when the dependency does not exist
    */
-  public MasterSource getMasterSource() {
-    return mMasterSource;
+  public DependencyInfo getClientDependencyInfo(int dependencyId)
+      throws DependencyDoesNotExistException {
+    return mFileSystemMaster.getClientDependencyInfo(dependencyId);
   }
 
   /**
-   * Get a new user id
+   * Gets the file info for the file at the given path
```

Comment:
Method descriptions should end with a '.' (applies to other methods in this file)